src/controllers/sessionController.ts

```diff
@@ -23,4 +31,23 @@ export default class SessionController {
         walletToken: user.wallet_token
       })
     );
+  public readonly listSessions = (
+    req: express.Request
+  ): Promise<
+    // tslint:disable-next-line: max-union-size
+    | IResponseErrorNotFound
+    | IResponseErrorInternal
+    | IResponseErrorValidation
```

Comment:
you should probably add status 400 [here](https://github.com/teamdigitale/io-backend/pull/432/files#diff-aa6cd0613984937586c55eb6ed53d7baR345)

Reply:
fixed in 3b7b721